priv/templates/freebsd.pkg/post_install.sh.eex

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+# GENERATED FILE! Do not modify.
+#
+# This script is run during `pkg install` after files
+# have been put in place.
+#
+
+CONFIG_DIR="<%= @config_dir %>"
```

Comment:
I wonder if these scripts should have their own RC commands in addition / instead. RC provides the env vars, so they shouldn't need to be written into the file.